sqlparser/seata-sqlparser-core/pom.xml

```diff
@@ -27,8 +27,15 @@
     <dependencies>
         <dependency>
             <groupId>${project.groupId}</groupId>
-            <artifactId>seata-common</artifactId>
+            <artifactId>seata-config-core</artifactId>
             <version>${project.version}</version>
         </dependency>
+
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>seata-core</artifactId>
+            <version>${project.version}</version>
+            <scope>provided</scope>
```

Comment:
why use `provided`  scope ?
In my opinion, `seata-core` is a self-managed module in the Seata project, not a dependency of a third party.

Reply:
done